code/modules/vehicles/scooter.dm

```diff
@@ -72,17 +100,52 @@
 	. = ..()
 	if(A.density && has_buckled_mobs())
 		var/mob/living/H = buckled_mobs[1]
-		var/atom/throw_target = get_edge_target_turf(H, pick(GLOB.cardinals))
-		unbuckle_mob(H)
-		H.throw_at(throw_target, 4, 3)
-		H.Paralyze(100)
-		H.adjustStaminaLoss(40)
-		var/head_slot = H.get_item_by_slot(SLOT_HEAD)
-		if(!head_slot || !(istype(head_slot,/obj/item/clothing/head/helmet) || istype(head_slot,/obj/item/clothing/head/hardhat)))
-			H.adjustOrganLoss(ORGAN_SLOT_BRAIN, 3)
-			H.updatehealth()
-		visible_message("<span class='danger'>[src] crashes into [A], sending [H] flying!</span>")
-		playsound(src, 'sound/effects/bang.ogg', 50, 1)
+		H.adjustStaminaLoss(instability*6)
+		playsound(src, 'sound/effects/bang.ogg', 40, 1)
+		if(!iscarbon(H) || H.getStaminaLoss() >= 100 || grinding || world.time - last_crash < 10)
+			var/atom/throw_target = get_edge_target_turf(H, pick(GLOB.cardinals))
+			unbuckle_mob(H)
+			H.throw_at(throw_target, 3, 2)
+			var/head_slot = H.get_item_by_slot(SLOT_HEAD)
+			if(!head_slot || !(istype(head_slot,/obj/item/clothing/head/helmet) || istype(head_slot,/obj/item/clothing/head/hardhat)))
+				H.adjustOrganLoss(ORGAN_SLOT_BRAIN, 5)
+				H.updatehealth()
+			visible_message("<span class='danger'>[src] crashes into [A], sending [H] flying!</span>")
+			H.Paralyze(80)
+		else
+			var/backdir = turn(dir, 180)
+			vehicle_move(backdir)
+			H.spin(4, 1)
+		last_crash = world.time
```

Comment:
instead of doing it like this, do `next_crash` which is world.time + 10 (or whatever the cooldown is)
then in every check just check world.time > next_crash, bit faster as the math op is only done once instead of every check, and also makes the cooldown only in one place.